Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -42,8 +42,8 @@ public async Task InitDefaultCustomCommands()
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s+(?<Roles>.*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s+(?<Roles>.*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s+(?<Roles>.*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.AddWarnCommand", @"-addwarn\s*-user\s*(?<User><@\d+>)\s*-(reason|r)\s+(?<Reason>.+)", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.WarnsCommand", @"-warns\s*(-user\s*(?<User><@&?\d+>))?\s*", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.AddWarnCommand", @"-addwarn\s*-user\s*(?<User><\D{1,2}\d+>)\s*-(reason|r)\s+(?<Reason>.+)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.WarnsCommand", @"-warns\s*(-user\s*(?<User><\D{1,2}\d+>))?\s*", serverId),
```

Comment:
takie custom commands nie mają sensu jeśli i tak wymagasz żeby zawsze pisać -addwarn -user <@12456>...
```suggestion
                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.AddWarnCommand", @"-add\s*warn\s+(?<User><\D{1,2}\d+>)\s+(?<Reason>.+)", serverId),
                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.WarnsCommand", @"-warns\s*(?<User><\D{1,2}\d+>)?", serverId),
```